cmdb/interface/rest_api/framework_routes/type_routes.py

```diff
@@ -168,8 +168,9 @@ def update_type(public_id: int, data: dict):
     """
     type_manager = TypeManager(database_manager=current_app.database_manager)
     try:
-        type_ = TypeModel.from_data(data=data)
 
+        data.setdefault('last_edit_time', datetime.utcnow())
```

Comment:
Wrong level for the default value. Put it in the constructor.
Also, please use `now` and not `utcnow`.
This is also written in the Python documentation:  https://docs.python.org/3/library/datetime.html

Reply:
We cannot simply call datetime.now(). Because this function returns the current local date and time. Instead of datetime.utcnow(). utcnow() returns the current UTC date and time, with tzinfo None. Therefore, we generally need to replace all datetime.now() with datetime.now(timezone.utc).

Reply:
We call `now()` in most places. If there are problems with the time zone, it is best to create a task for this so that it is kept consistent. It is only important that a datetime object is passed on to the database so that it recognises the field as a date.

Reply:
Ok, I see you already started that.